apps/api/src/form/form.controller.ts

```diff
@@ -43,4 +50,39 @@ export class FormController {
   async getFormById(@Param('id') id: string): Promise<FormEntity> {
     return await this.formService.getFormById(id);
   }
+
+  @ApiOperation({
+    summary: 'Export Data',
+  })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @ApiResponse({ status: HttpStatus.CREATED, type: EmptyResponse })
+  @HttpCode(HttpStatus.CREATED)
+  @Get('/export/:passCode')
+  async exportAll(@Param('passCode') passCode: string, @Res() res: Response) {
```

Comment:
I think this is more logic than I'd prefer in a controller, can this be made into a service file?

Reply:
I don't think so. I initially had a csv generator module, and it's very little functionality to have it's own service. 

Also we need to do the filtering and sorting here anyways.